app/controllers/api/v1/events_controller.rb

```diff
@@ -4,22 +4,24 @@ module Api
   module V1
     class EventsController < Api::BaseController
       def create
-        # NOTE: Properties validations will be returned on the debugger.
-        validate_result = Events::CreateService.new(
+        result = ::Events::CreateService.new(
           organization: current_organization,
-        ).validate_params(
+        ).call(
           params: create_params,
+          timestamp: Time.current.to_f,
+          metadata: event_metadata,
```

Comment:
What about passing `organization` inside the call method as well?

Something like `Events::CreateService.call(...)`